package.json

```diff
@@ -1,83 +1,31 @@
 {
-  "name": "styled-components",
+  "name": "styled-components-project",
+  "private": true,
   "version": "4.1.3",
-  "description": "Visual primitives for the component age. Use the best bits of ES6 and CSS to style your apps without stress",
-  "main": "dist/styled-components.cjs.js",
-  "jsnext:main": "dist/styled-components.esm.js",
-  "module": "dist/styled-components.esm.js",
-  "react-native": "native/dist/styled-components.native.cjs.js",
-  "browser": {
-    "./dist/styled-components.esm.js": "./dist/styled-components.browser.esm.js",
-    "./dist/styled-components.cjs.js": "./dist/styled-components.browser.cjs.js"
-  },
-  "sideEffects": false,
-  "scripts": {
-    "build": "rollup -c",
-    "prebuild": "rimraf dist",
+  "scripts":{
+    "bs": "lerna bootstrap",
+    "build": "lerna run build --stream --parallel",
+    "clean":"lerna clean",
+    "dev": "lerna run dev --stream --parallel",
+    "flow": "lerna run flow --stream --parallel",
+    "lint": "lerna run lint --stream --parallel",
+    "lint-staged": "lerna run lint-staged --stream --parallel",
```

Comment:
I don't think lint-staged should run once per package, this should run on the entire repo, right?

Reply:
Ah. That's true.